engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -271,20 +261,19 @@ private void preRenderUpdate(RenderingStage renderingStage) {
         resetStats();
 
         currentRenderingStage = renderingStage;
-        if (currentRenderingStage == RenderingStage.MONO || currentRenderingStage == RenderingStage.LEFT_EYE) {
-            isFirstRenderingStageForCurrentFrame = true;
-        } else {
-            isFirstRenderingStageForCurrentFrame = false;
-        }
+        isFirstRenderingStageForCurrentFrame = (currentRenderingStage == RenderingStage.MONO) || (currentRenderingStage == RenderingStage.LEFT_EYE);
```

Comment:
I mentioned this already: I prefer the IF block as it was before as it's a crucial piece of code and it needs its "visual space."